test/index.test.js

```diff
@@ -95,3 +96,12 @@ test("should handle User type and isAdult function", async (t) => {
 	const { code } = transformSync(inputCode);
 	t.assert.snapshot(code);
 });
+
+test("should handle empty source code", (t) => {
+	assert.strictEqual(transformSync().code, "");
+	assert.strictEqual(transformSync({}).code, "");
+	assert.strictEqual(transformSync(false).code, "");
```

Comment:
Seems like the change was dropped.

```suggestion
	assert.strictEqual(transformSync(false).code, "false");
```

Reply:
test pass, so its behaving correctly